TestVectors/dafny/TestVectors/src/VectorsComposition/AllEsdkV5NoReqEc.dfy

```diff
@@ -0,0 +1,108 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+include "../LibraryIndex.dfy"
+
+module {:options "/functionSyntax:4"} AllEsdkV5NoReqEc {
```

Comment:
`EsdkV5`? Do we have `Esdk V5`?

Reply:
I think it mean manifest v5 not ESDK V5?

Reply:
I think the context was the TestVectors.